.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "no-unused-vars": "off"
```

Comment:
In case anybody is wondering why.

It works fine except for some false positives with `no-unused-vars` caused by imported types. 
The solution is to use [`standardx`](https://github.com/standard/standardx) for custom eslint rule tweaking support. And disable `no-unused-vars` base rule as [instructed by `eslint-plugin-typescript`](
https://github.com/bradzacher/eslint-plugin-typescript/blob/master/docs/rules/no-unused-vars.md#options).